### PR TITLE
Create xs avatar component

### LINKED ISSUE #185: ⚛️ Create XSAvatar component

### PR DESCRIPTION
**Pull request by fizcamposancos**
Small Avatar is now medium and the new xs is called small

closes #185

### CODE REVIEW

File: webapp/src/pages/LoginPage/LoginPage.tsx [hunk 1]
```diff
@@ -14,9 +14,13 @@ import {
 import { ResponseError } from '../../shared/generated';
 import { authApi } from '../../shared/services/ApiService';
 import { LOGIN_EP_URL, REGISTER_URL, USERS_URL } from '../../shared/urls';
-import { SubmitStatus } from '../../shared/types';
 import './LoginPage.css';
 
+type SubmitStatus = {
```
Comment (Damrod) [inline webapp/src/pages/LoginPage/LoginPage.tsx:19]: every MR is changing this file, so you'll probably have conflicts. But you'll figure it out, I'm sure.

File: webapp/src/index.tsx [hunk 2]
```diff
@@ -40,7 +38,6 @@ root.render(
         <Routes>
           <Route path="/" element={<LandingPage />} />
           <Route path={LOGIN_OPTIONS_URL} element={<LoginPage />} />
-          <Route path={REGISTER_URL} element={<RegisterPage />} />
```
Comment (Damrod) [inline webapp/src/index.tsx:43]: why do you remove the register page in a PR the scope of which is limited to adding a component?

File: webapp/src/shared/components/Avatar/Avatar.tsx [hunk 3]
```diff
@@ -17,7 +17,8 @@ type LargeAvatarProps = AvatarProps & {
   editUrl?: string;
 };
 
-const SMALL_AVATAR_DOWNSCALE = 0.2;
+const SMALL_AVATAR_DOWNSCALE = 0.05;
```
Comment (Damrod) [inline webapp/src/shared/components/Avatar/Avatar.tsx:20]: I would like to double check this constant, but not now. It appears good, so I'l take it as such.

**Review by Damrod — APPROVED** [review]
besides the comments, LGTM

File: webapp/src/pages/LoginPage/LoginPage.tsx (hunk 1 above)
Comment (fizcamposancos) [reply webapp/src/pages/LoginPage/LoginPage.tsx:19]: Yes, and sorry, It was a little mess but I will be careful when merging

File: webapp/src/index.tsx (hunk 2 above)
Comment (fizcamposancos) [reply webapp/src/index.tsx:43]: Sorry, same as before, I tried to focus on this PR but starting from another branch, but I mess everything a little. I will be careful when merging

File: webapp/src/shared/components/Avatar/Avatar.tsx (hunk 3 above)
Comment (fizcamposancos) [reply webapp/src/shared/components/Avatar/Avatar.tsx:20]: Yes, do it, please. I've just extrapolate from the ones you put

**Review by vdedios — APPROVED** [review]
LGTM! 🚀

**Review by gbudau — APPROVED** [review]
LGTM :tada:

**Pull request merged**